src/System.Private.Xml/src/System/Xml/XPath/Internal/XPathArrayIterator.cs

```diff
@@ -84,7 +84,5 @@ public override IEnumerator GetEnumerator()
         {
             return list.GetEnumerator();
         }
-
-        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
```

Comment:
How does removing this flag it as a bad merge?

I'm unclear as to the purpose of this PR.  Did you intend to get it into a state where we could simply hit the Merge button in GitHub?  Or was your goal here just to highlight code that could potentially be deleted but expect this PR to be closed rather than merged?

Thanks.